client/src/test/scala/dwolla/cloudflare/RateLimitClientSpec.scala

```diff
@@ -11,12 +12,15 @@ import org.specs2.specification.Scope
 
 class RateLimitClientSpec extends Specification {
   trait Setup extends Scope {
-    val zoneId = "zone-id"
+    val zoneId: ZoneId = "zone-id"
 
     val authorization = CloudflareAuthorization("email", "key")
     val fakeService = new FakeCloudflareService(authorization)
   }
 
+  implicit val tagZoneId: String ⇒ ZoneId = shapeless.tag[ZoneIdTag][String]
```

Comment:
I think these implicits are likely to be counterproductive here. They effectively allow one to use any string anywhere a `ZoneId` or `RateLimitId` is called for, but the reason we're introducing those tagged types is to prevent just any string from being used in those places. In other words, these implicits reintroduce the possibility of the kinds of bugs the tagged types are meant to prevent. 🙁

I would probably change line 15 to 

```scala
val zoneId = shapeless.tag[ZoneIdTag][String]("zone-id")
val rateLimitId = shapeless.tag[RateLimitIdTag][String]("rate-limit-id")
```

remove the implicits on lines 21-22, and then use those provided example values throughout the tests.

Reply:
Added explicit methods because I needed more than one RateLimitId throughout the test.